lib/VerifiableCredentialStore.js

```diff
@@ -167,7 +167,12 @@ export default class VerifiableCredentialStore {
 
       // build query to find all VCs that match any combination of type+issuer
       const query = [];
-      const issuers = trustedIssuers.map(({id}) => id);
+      const issuers = trustedIssuers.map(({id}) => {
+        if(!id) {
+          throw new Error('"id" must be a string.');
```

Comment:
I have two concerns here:

1. What is the UX when this error is thrown?
2. We need to handle cases where no `id` is set in the future, but other attributes are provided (such as a nested VC query indicating a chain of trust use case, e.g., where any issuer with a VC that says they are an accredited university  will be trusted). We don't have to handle this now, but the error should probably indicate that the query is unsupported vs. it being invalid.

Reply:
1. So, the error on the UX currently is the same as when a non-matching trustedIssuer is sent in query - It says `Sorry, we can't seem to find any credentials.` 
2. I've updated the error message to indicate that a trustedIssuer without an id is unsupported.